persistence-common/src/main/java/io/stargate/db/datastore/common/AbstractCassandraPersistence.java

```diff
@@ -13,18 +20,15 @@
  * between the persistence layers for various C* versions.
  *
  * @param <Config>
- * @param <ClientState>
- * @param <QueryState>
  * @param <K> the concrete class for keyspace metadata in the persistence layer.
  * @param <T> the concrete class for table metadata in the persistence layer.
  * @param <C> the concrete class for column metadata in the persistence layer.
  * @param <U> the concrete class for user types in the persistence layer.
  * @param <I> the concrete class for secondary indexes metadata in the persistence layer.
  * @param <V> the concrete class for materialized views metadata in the persistence layer.
  */
-public abstract class AbstractCassandraPersistence<
-        Config, ClientState, QueryState, K, T, C, U, I, V>
-    implements Persistence<Config, ClientState, QueryState> {
+public abstract class AbstractCassandraPersistence<Config, K, T, C, U, I, V>
```

Comment:
Not related to this changeset, but can we rename `schemaConverter()` method to `newSchemaConverter()` ?